indexer/services/ender/__tests__/lib/on-message.test.ts

```diff
@@ -239,6 +252,73 @@ describe('on-message', () => {
       expect.any(Number), 1, { success: 'true' });
   });
 
+  it('successfully processes block with market create and funding events', async () => {
```

Comment:
nit: update name of the test to indicate the funding events are for the created market.

Reply:
done